infrastructure/terraform/components/reporting/module_powerbi_gateway_vpc.tf

```diff
@@ -0,0 +1,60 @@
+module "powerbi_gateway_vpc" {
+
+  source  = "terraform-aws-modules/vpc/aws"
+  version = "5.5.1"  # Adjust to the latest version
+
+  create_vpc = var.enable_powerbi_gateway
+
+  name = "${local.csi}-powerbi-gateway-vpc"
+  cidr = "10.0.0.0/16"
+
+  azs             = var.availability_zones
```

Comment:
I think we can do this a little better to let terraform handle it, rather than define AZs ourselves, we could just have a data source give us the ones in the region because we would just future proof it that way: https://registry.terraform.io/providers/hashicorp/aws/latest/docs/data-sources/availability_zones

Reply:
done.